orca/orca-echo/src/main/groovy/com/netflix/spinnaker/orca/echo/spring/EchoNotifyingStageListener.groovy

```diff
@@ -144,7 +144,7 @@ class EchoNotifyingStageListener implements StageListener {
         MDC.put(Header.EXECUTION_ID.header, stage.execution.id)
         MDC.put(Header.USER.header, stage.execution?.authentication?.user ?: "anonymous")
         AuthenticatedRequest.allowAnonymous({
-          Retrofit2SyncCall.execute(echoService.recordEvent(event))
+          Retrofit2SyncCall.execute(echoService.recordEvent(event as Map))
```

Comment:
Curious about the casts here too, but I can probably close my eyes.  One day we'll convert this to java...

Reply:
Again, groovy's map literal type inference is more flexible with wildcards. So when I replaced wildcard with specific type, groovy complained.